src/main/scala/nl.knaw.dans.easy.multideposit/actions/AddPropertiesToDeposit.scala

```diff
@@ -58,7 +58,8 @@ class AddPropertiesToDeposit extends DebugEnhancedLogging {
       "springfield.domain" -> sf.map(_.domain),
       "springfield.user" -> sf.map(_.user),
       "springfield.collection" -> sf.map(_.collection),
-      "springfield.playmode" -> sf.map(_.playMode.toString)
+      "springfield.playmode" -> sf.map(_.playMode.toString),
+      "identifier.doi.registered" -> Some("no")
```

Comment:
Please add a comma after this last item. Next time around the diff will be smaller.

Reply:
will do